sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -41,18 +46,107 @@ case class WriteToDataSourceV2(batchWrite: BatchWrite, query: LogicalPlan)
   override def output: Seq[Attribute] = Nil
 }
 
+case class AppendDataExec(
+    table: SupportsBatchWrite,
+    writeOptions: DataSourceOptions,
+    query: SparkPlan) extends V2TableWriteExec with BatchWriteHelper {
+
+  override protected def doExecute(): RDD[InternalRow] = {
+    val batchWrite = newWriteBuilder() match {
+      case builder: SupportsSaveMode => // TODO: Remove this
```

Comment:
It seems that we have three TODOs here. Shall we create SPARK issue and use the ID here explicitly? I guess that all three TODO will have the same SPARK issue ID.

Reply:
Wenchen already opened an issue for this: [SPARK-26356](https://issues.apache.org/jira/browse/SPARK-26356)

Reply:
Yes. That's what I requested. We had better have a IDed TODO.
```
// TODO: Remove this
// TODO(SPARK-26356): Remove this
```

Reply:
I'm removing the comments. If SupportsSaveMode is no longer defined, then this has to be removed anyway. The comment doesn't help.